conftest.py

```diff
@@ -122,3 +123,36 @@ def pytest_configure(config):
     #
     # Somehow the default database is always in memory, though.
     settings.DATABASES['transient']['TEST_NAME'] = ':memory:'
+
+
+@pytest.yield_fixture()
+def installed_packages_catalog():
```

Comment:
> Also I generally tend to use mock for I/O (that we cannot reproduce during tests), but avoid them for internal code.

It is not unittest then. If I want to test a function depending of a module, I have to mock the module, not the dependencies of the module.

However, I agree that it mocks a lot of thing, especially creating packages that we may not want. I will try to rewrite it to have a better API.